ProductsWebApi/ProductsWebApi/Validator/UserValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProductsWebApi.Validator
+{
+    public class UserValidator : AbstractValidator<Users>
```

Comment:
so, you have used Fluent Validations before?

Reply:
I have gone through a few articles and tried to implement them basically.